scripts/test_schemas.sh

```diff
@@ -6,8 +6,8 @@ if [ "$1" == "--local" ] || [ "$2" == "--local" ]; then
 fi
 
 if [ "$run_docker" == true ]; then
-    docker pull onsdigital/eq-schema-validator
-    validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
```

Comment:
Will need reverting once [validator PR](https://github.com/ONSdigital/eq-schema-validator/pull/55) has been approved. Both should go in together.